### PR TITLE
refactor(frontend): migrate 6 tests from Enzyme to RTL

### PR DESCRIPTION
**Pull request by hainenber**
refactor(frontend): migrate 6 tests from Enzyme to RTL

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by mistercrunch** [comment]
Nice! please tag us when this becomes ready for review

**Comment by hainenber** [comment]
This is ready for review :D

File: superset-frontend/src/components/FacePile/FacePile.test.tsx [hunk 1]
```diff
@@ -29,23 +28,37 @@ const users = [...new Array(10)].map((_, i) => ({
   id: i,
 }));
 
+jest.useFakeTimers();
```
Comment (justinpark) [inline superset-frontend/src/components/FacePile/FacePile.test.tsx:31]: Please set it back to real timer after all specs are finished.

```suggestion
beforeEach(() => {
  jest.useFakeTimers();
});

afterEach(() => {
  jest.useRealTimers();
});
```

File: superset-frontend/src/explore/components/controls/SelectControl.test.jsx [hunk 2]
```diff
@@ -30,139 +37,198 @@ const defaultProps = {
...
-            />,
-          );
-          expect(withMulti.html()).not.toContain('option(s');
+          const container = renderSelectControl({
+            choices: [],
+            multi: true,
+            placeholder: 'add something',
+          });
+          expect(
+            container.querySelector('[role="option"]'),
```
Comment (justinpark) [inline superset-frontend/src/explore/components/controls/SelectControl.test.jsx:178]: queryByRole is better for this case.
```suggestion
            queryByRole('option')
```

**Review by justinpark — APPROVED** [review]
LGTM

**Pull request merged**